src/model/procedure/FluxProcedureRequested.js

```diff
@@ -75,6 +75,14 @@ class FluxProcedureRequested {
         }
     }
 
+    get expectedPerformer() {
+        if (this._procedureRequested.expectedPerformer && this._procedureRequested.expectedPerformer.value && this._procedureRequested.expectedPerformer.value.person && this._procedureRequested.expectedPerformer.value.person) {
```

Comment:
`this._procedureRequested.expectedPerformer.value.person` is checked twice.  One of these can probably be removed unless you also want to check that `this._procedureRequested.expectedPerformer.value.person.name` exists.